addon/components/nypr-story/header-content.js

```diff
@@ -0,0 +1,21 @@
+import Ember from 'ember';
+import computed from 'ember-computed';
+import layout from '../../templates/components/nypr-story/header-content';
+
+export default Ember.Component.extend({
+  layout,
+  pk: computed('story.id', function() {
+    // coerce to string b/c ember-data expects
+    // a string for IDs
+    return String(get(this, 'story.id'));
+  }),
+
+  didRender() {
```

Comment:
[the `didInsertElement` hook is the place to call third-party JS libs](https://guides.emberjs.com/v2.13.0/components/the-component-lifecycle/#toc_integrating-with-third-party-libraries-with-code-didinsertelement-code)